components/SingleSelectRadio/styles.ts

```diff
@@ -0,0 +1,40 @@
+import styled, { css } from 'styled-components';
+import COLORS from '@/styles/colors';
+
+interface SelectedProps {
+  isSelected: boolean;
```

Comment:
all props in StyledComponents should start with a $, so this should be `$isSelected` here and elsewhere